### DISCUSSION TITLE
citekeys without brackets not converted to Zotero live citations with zotero.lua

### DISCUSSION THREAD

**Discussion by trickard1000** [retorquere/zotero-better-bibtex#2425] [Q&A] [answered]
Hi.

My understanding was that without brackets, citekeys would become in line citations - i.e.,:

"@Rickard_1993 stated that this was a rabbit hole" becomes "Rickard (1993) stated that this was a rabbit hole".

This does not work

----------------------------------

I run pandoc with the zotero.lua filter and library.yaml to produce a word doc with live zotero citations.

The [@citekeys] in brackets are converted to live citations. 

The @citekeys without brackets are not converted.

---------------------------------

Can you help at all? My thanks if you can.

Windows 10
pandoc 3.1
Zotero 6.0.22, Better Bibtex installed
Word 365 desktop

**Comment by retorquere** [comment, marked as answer]
Zotero doesn't actually support in-text citations, so the lua filter will just leave them in the body by default. If you set [author-in-text](https://retorque.re/zotero-better-bibtex/exporting/pandoc/#from-markdown-to-zotero-live-citations) to `true` the script will try to fake support for in-text citations by adding the authors to the body, and adding a suppressed-author citation after it.
